week_1/task_3/Task_3.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <cmath>
+int main()
+{
+    double a;
+    double b;
+    double c;
+
+    std::cin >> a >> b >> c;
+
+    if ((b * b - 4 * a * c) > 0) {
```

Comment:
@Ypostolov if b = 0 and c = 0

Reply:
Fixed in 14541a0056ce85b5708ffbe8367445899047bc35.
Please check.

Reply:
Fixed in 14541a0056ce85b5708ffbe8367445899047bc35.
Please check.